UI/views/todo.html

```diff
@@ -1,11 +1,11 @@
 <div ng-controller="todoController" ngcloak>
-    <div id="todo" class="itemPanel">
-      <h3 class="colHeader"><span class="sr-only">To Dos</span>
+    <div id="schedule" class="itemPanel">
+      <h3 class="colHeader"><span class="sr-only">Schedule</span>
```

Comment:
I noticed that the div id was changed from todo to schedule, but the controller name remains todoController. Does it make more sense for the controller to be named scheduleController?